Lesson22/ezgi/app.js

```diff
@@ -0,0 +1,43 @@
+const formElement = document.getElementById('toDoForm');
+const formInput = document.getElementById('toDoInput');
+const list = document.getElementById('toDoList');
+
+formElement.addEventListener('submit', function (event) {
+	// By default, the page is reloded when form is submitted. To avoid it, we need to add `preventDetault()`;
+	event.preventDefault();
+
+  // TODO: check that input value has correct format (e.g. minimum 3 chars, no extra spaces)
+
+  const inputValue = formInput.value.trim();
+	const firstLetterUpperCase = inputValue.charAt(0).toUpperCase() + inputValue.slice(1);
+
+	if (inputValue.length < 3) {
+		alert("Please enter a todo with at least 3 characters.");
```

Comment:
Using alert messages is considered bad user experience. Instead, in cases like this you can show error message by adding text of an HTML element. E.g. add an empty <p> element to HTML just below the form:
`<p id="errorMsg"></p>`

Then here in JS set its inner text to whatever text message you want to show: 
`document.getElementById('errorMsg').innerText = "Please enter a todo with at least 3 characters.";`
You may also need to clear content of the errorMsg element when it is not relevant any more, you can do it by setting its innerText to empty string:
`document.getElementById('errorMsg').innerText ="";`